funding/manager.go

```diff
@@ -64,12 +64,12 @@ const (
 	// to use for its commitment transaction.
 	MaxBtcRemoteDelay uint16 = 2016
 
-	// MinLtcRemoteDelay is the minimum Litecoin CSV delay we will require the
-	// remote to use for its commitment transaction.
+	// MinLtcRemoteDelay is the minimum Litecoin CSV delay we will require
```

Comment:
thanks for all the clean up! 
one suggestion I have for future PRs though is to maybe do the cleanups in separate commits from the logic changes just so that it is easier to parse these diffs :)

Reply:
Yes I agree, I wasn't thinking about it from a reviewer perspective when I did it, now it's kinda cumbersome to revert the cleaned up lines.

If you think it's definitely necessary to revert let me know and I will do it. Will keep it in mind for future commits.

Reply:
Done so